cmd/serverless/main.go

```diff
@@ -210,27 +227,16 @@ func runAgent(stopCh chan struct{}) (daemon *serverless.Daemon, err error) {
 		}
 	}
 
-	// read configuration from both the environment vars and the config file
-	// if one is provided
-	// --------------------------
-
-	aws.SetColdStart(true)
-
-	config.Datadog.SetConfigFile(datadogConfigPath)
-	if _, confErr := config.LoadWithoutSecret(); confErr == nil {
-		log.Info("A configuration file has been found and read.")
-	}
-
 	// adaptive flush configuration
 	if v, exists := os.LookupEnv(flushStrategyEnvVar); exists {
 		if flushStrategy, err := flush.StrategyFromString(v); err != nil {
 			log.Debugf("Wrong flush strategy %s, will use the adaptive flush instead. Err: %s", v, err)
```

Comment:
I know it's not new in this PR but might as well fix this
```suggestion
			log.Debugf("Invalid flush strategy %s, will use adaptive flush instead. Err: %s", v, err)
```